server/config/base.yaml

```diff
@@ -0,0 +1,2 @@
+temporalGrpcAddress: 127.0.0.1:7233
+port: 8080
```

Comment:
since the UI assets now target :8233 in local dev mode let's also change this port number to 8233 so we don't have to modify each time we want to run the ui server from code